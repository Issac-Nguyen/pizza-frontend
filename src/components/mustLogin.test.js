import React from 'react'
import {render, cleanup} from '@testing-library/react'
import MustLogin from './mustLogin'

describe('mustLogin', () => {
    afterEach(cleanup);
 
    it('should take a snapshot', () => {
       const { asFragment } = render(<MustLogin />)
       
       expect(asFragment(<MustLogin />)).toMatchSnapshot()
    })

    it('should have h1', () => {
        const { getByText } = render(<MustLogin />)
        
        expect(getByText(/you need/i)).toBeInTheDocument();
     })
})
