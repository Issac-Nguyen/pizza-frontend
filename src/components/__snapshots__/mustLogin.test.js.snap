// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mustLogin should take a snapshot 1`] = `
<DocumentFragment>
  <h1>
    You need to login to see this feature
  </h1>
</DocumentFragment>
`;
